examples/rbac/server/server_resources.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: feast-sa
+  namespace: feast-dev
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: feast-cluster-binding
+  namespace: feast-dev
+subjects:
+  - kind: ServiceAccount
+    name: feast-sa
+    namespace: feast-dev
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
`cluster-admin` we don't required we required the service account to list the roles for users so instead we use roles for that.

if we don't use new service account then the role will be bind to``default` service account.  I just create our service account instead of default one.

Reply:
create the new roles which can list ["roles", "rolebindings", "clusterrolebindings"] and bind to `feast-sa` service account.